Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
```

Comment:
### [dotenv](https://github.com/bkeepers/dotenv)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.8.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/dotenv/2.2.1/2.8.1)_]





<details>
<summary>Commits</summary>

A change of **183** commits. See the full changes on [the compare page](https://github.com/bkeepers/dotenv/compare/v2.2.1...v2.8.1).

These are the first 10 commits:
- (697865b) [Skip over lines that don't set an envvars](https://github.com/bkeepers/dotenv/commit/697865b86bcc2c4cc33c09dd516d02c1ba28e5a8)
- (6e885a1) [Changes script type](https://github.com/bkeepers/dotenv/commit/6e885a18bc3193283b7ae4b90abcfa37768352a5)
- (a47020f) [Merge pull request #298 from edersonbadeca/change-read-me](https://github.com/bkeepers/dotenv/commit/a47020f6c414e0a577680b324e61876a690d2200)
- (3d0b424) [Document the -f flag](https://github.com/bkeepers/dotenv/commit/3d0b4245ceb1f88ca6c09274fe9f5aa60e55d15f)
- (a689e6a) [Existing variable takes precendence when performing substitu…](https://github.com/bkeepers/dotenv/commit/a689e6ae68b99b88626e5fe9e675dd17b730167b)
- (a7fd9b5) [CI against Ruby 2.5.0](https://github.com/bkeepers/dotenv/commit/a7fd9b59306b362f0183ccd02151d65a048266fc)
- (c7d6342) [Parse multi-line values properly (e.g. Heroku config export)](https://github.com/bkeepers/dotenv/commit/c7d634251f42b9d6f80f888e469a52f727c5a9a5)
- (c9c82b0) [Fix original regex to escape slash](https://github.com/bkeepers/dotenv/commit/c9c82b025014fcd1dc3484ce175acbd429ae4a60)
- (694c566) [Match multiple lines](https://github.com/bkeepers/dotenv/commit/694c566da91acf9768daf6ecd2334f3572ba0de2)
- (3bb0357) [Add tests for parsing multi-line values](https://github.com/bkeepers/dotenv/commit/3bb0357a07a7e76d485733e18a71f03aeddce32b)


</details>